Manifests/ManagedPreferencesApplications/menu.nomad.login.ad.plist

```diff
@@ -262,6 +289,35 @@
 					</dict>
 				</array>
 			</dict>
+			<dict>
+				<key>pfm_app_min</key>
+				<string>1.4+</string>
```

Comment:
pfm_app_min just has to have the first version that supports the preference, so `1.4` is sufficient.  The app adds the +.